wolfcrypt/src/ed25519.c

```diff
@@ -229,25 +243,24 @@ int wc_ed25519_sign_msg_ex(const byte* in, word32 inLen, byte* out,
     az[31] &= 63; /* same than az[31] &= 127 because of az[31] |= 64 */
     az[31] |= 64;
 
-    ret = wc_InitSha512_ex(&sha, NULL, devId);
+    ret = ed25519_hash_reset(key);
```

Comment:
Doesn't appear necessary to make this change.

Reply:
I assume you mean the `_reset()` isn't necessary.  You're absolutely right.  The earlier call to `ed25519_hash()` assures that the hash is in a known-clean state at that codepoint.